### PR TITLE
Use table listing in `History of osu!`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Continuation of #9055, and therefore requires it before merge.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] Depends on https://github.com/ppy/osu-wiki/pull/9055

### CODE REVIEW

File: wiki/History_of_osu!/en.md [hunk 1]
```diff
@@ -7,18 +7,23 @@ needs_cleanup: true
...
+| Year | Description |
+| :-: | :-- |
+| [2007](2007) | Initial game build |
+| [2008](2008) | Taiko and Catch the Beat, more mods, and major improvements |
+| [2009](2009) | osu! iPhone |
+| 2010 | *Undocumented* |
+| [2011](2011) | osu! wiki |
+| [2012](2012) | Five years of osu!, osu!mania, widescreen support, and visual settings |
+| [2013](2013) | UI overhaul, HD skins, improved widescreen support, osu!mania editor, in-game collections, moddingV2 was introduced, Disqus, osu!api, accuracy overlays, Comic Fiesta 2013, downloads unrestricted, and osu!cuttingedge (osu!supporters only) |
+| [2014](2014) | BPM scrolling, in-game intro/outro sequences, osu!cuttingedge (for everyone), and snow! |
```
Comment (Walavouchey) [inline wiki/History_of_osu!/en.md:19]: only thing i can think of for "snow!" that would make sense, but the article doesn't mention it

```suggestion
| [2014](2014) | osu!mania BPM scrolling, in-game intro/outro sequences, osu!cuttingedge (for everyone), and main menu snow |
```

File: wiki/History_of_osu!/en.md [hunk 2]
```diff
@@ -7,18 +7,23 @@ needs_cleanup: true
...
+| [2015](2015) | jizz (redesigned osu!web), osu!weekly, and osu!coins |
```
Comment (Walavouchey) [inline wiki/History_of_osu!/en.md:20]:
```suggestion
| [2015](2015) | osu!web redesign, osu!weekly, and osu!coins |
```

File: wiki/History_of_osu!/en.md [hunk 3]
```diff
@@ -7,18 +7,23 @@ needs_cleanup: true
...
+| [2016](2016) | scorev2, announcement of a new open-source "lazer" version of osu! (previously known as "osu!lazer"), and an osu! wiki overhaul |
+| [2017](2017) | release of the first lazer build of osu!, and Discord integration on the stable client |
+| [2018](2018) | osu!next, depreciation of the old website, total play time display |
```
Comment (Walavouchey) [inline wiki/History_of_osu!/en.md:23]:
```suggestion
| [2018](2018) | osu!next, deprecation of the old website, total play time display |
```

File: wiki/History_of_osu!/en.md [hunk 4]
```diff
@@ -7,18 +7,23 @@ needs_cleanup: true
...
+| 2019–2021 | *Undocumented* |
+| [2022](2022) | 15 years of osu!, new osu!monthly |
```
Comment (Walavouchey) [inline wiki/History_of_osu!/en.md:25]:
```suggestion
| [2022](2022) | 15 years of osu!, new osu!monthly news post series |
```

File: wiki/History_of_osu!/en.md [hunk 5]
```diff
@@ -7,18 +7,23 @@ needs_cleanup: true
...
-- [2013](2013) - UI overhaul, HD skins, improved widescreen support, osu!mania editor, in-game collections, moddingV2 was introduced, Disqus, osu!api, accuracy overlays, Comic Fiesta 2013, downloads unrestricted, and osu!cuttingedge (osu!supporters only)
-- [2014](2014) - BPM scrolling, in-game intro/outro sequences, osu!cuttingedge (for everyone), and snow!
-- [2015](2015) - jizz (redesigned osu!web), osu!weekly, and osu!coins
-- [2016](2016) - scorev2, announcement of a new open-source "lazer" version of osu! (previously known as "osu!lazer"), and an osu! wiki overhaul
-- [2017](2017) - release of the first lazer build of osu!, and Discord integration on the stable client
-- [2018](2018)–present - ?
+| Year | Description |
+| :-: | :-- |
+| [2007](2007) | Initial game build |
+| [2008](2008) | Taiko and Catch the Beat, more mods, and major improvements |
```
Comment (Walavouchey) [inline wiki/History_of_osu!/en.md:13]:
```suggestion
| [2008](2008) | osu!taiko and osu!catch (then Taiko and Catch the Beat), more mods, and major improvements |
```

Comment (The-Last-Cookie) [reply wiki/History_of_osu!/en.md:13]: I think it is better to move the name change explanation to the article (probably as a notice) and just call them their current name here.

File: wiki/History_of_osu!/en.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/History_of_osu!/en.md:19]: Should I add an issue or a to-do for this in the history article? I think adding a small text about it in the article is not much work.

Comment (Walavouchey) [reply wiki/History_of_osu!/en.md:19]: since it's just some random client feature i'll remove the mention of it for now

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**